.azuredevops/platformPipelines/platform.dependencies.yml

```diff
@@ -884,3 +884,18 @@ stages:
             - path: $(dependencyPath)/$(resourceType)/parameters/parameters.json
               templateFilePath: $(templateFilePath)
               displayName: Default Virtual Machine
+
+  - stage: deploy_lb
```

Comment:
Why the dependency? I don't see that we use it anywhere and I'd vote against adding it 'just in case we might use it someday'

Reply:
Should probably not have deployed it here in this PR. But I am using it in the testing of the addition that's introduced in this PR: #1092